tests/resources/Harbor-Pages/Project-Helmcharts.robot

```diff
@@ -37,6 +37,18 @@ Go Into Chart Detail
     Retry Element Click  xpath=//hbr-helm-chart-version//a[contains(., '${version_name}')]
     Retry Wait Until Page Contains Element  ${chart_detail}
 
+Download Chart File
+    [Arguments]  ${chart_name}  ${chart_filename}  ${download_directory}
+    Switch To Project Charts
+    ${out}  Run Keyword And Ignore Error  OperatingSystem.File Should Not Exist  ${download_directory}/${chart_filename}
+    Run Keyword If  '${out[0]}'=='FAIL'  Run  rm -rf ${download_directory}/${chart_filename}
+    OperatingSystem.File Should Not Exist  ${download_directory}/${chart_filename}
+    Retry Element Click  //clr-dg-row[contains(.,'${chart_name}')]//label
+    Retry Element Click  ${download_chart_button}
```

Comment:
Use "Retry Double Keywords When Error" to make sure that if file is not existed in download directory, we should try to click download button for  retry times.